vyxal/helpers.py

```diff
@@ -74,6 +80,12 @@ def deep_copy(value: Any) -> Any:
     return LazyList(itertools.tee(value)[-1])
 
 
+def dict_to_list(dictionary: dict) -> List[Any]:
+    """Returns a dictionary as [[key, value]]"""
+
+    return [[str(key), dictionary[key]] for key in dictionary]
```

Comment:
I think it should be `vyxalify`?

Reply:
This is because it's for converting Sympy solutions for an equation (which are `{symbol: value}` pairs) to a list of lists.

The str(key) is so that the key isn't left as a symbol.